apps/generator/test/generator.test.js

```diff
@@ -209,8 +211,8 @@ describe('Generator', () => {
 
       expect(xfsMock.mkdirpSync).toHaveBeenCalledWith(__dirname);
       expect(gen.installTemplate).toHaveBeenCalledWith(true);
-      expect(gen.loadTemplateConfig).toHaveBeenCalled();
-      expect(gen.loadTemplateConfig).toHaveBeenCalled();
+      expect(templateConfigLoader.loadTemplateConfig).toHaveBeenCalled();
```

Comment:
Weird that we have this duplicated, same in line 238-39
I did it in my PR but don't remember why :D did you try to change it and test once?